src/objectsAdvanced/task3.js

```diff
@@ -0,0 +1,5 @@
+const fn = (objectA, objectB) => {
+    const object = { ...objectA, ...objectB }
```

Comment:
The solution is right, but ... the function doesn't return any results!
And another hint: if function body has only one line you can omit curly brackets.
Try to improve.